airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -761,6 +761,12 @@ def run_query(self,
                 }
         :type encryption_configuration: dict
         """
+        # To provide backward compatibility
+        if isinstance(schema_update_options, tuple):
+            schema_update_options = list(schema_update_options)
+
+        if schema_update_options is None:
+            schema_update_options = []
```

Comment:
you probs dont need to check
```list([1,2,3]) == list((1,2,3))
True```

this would save a line and keeps all the same code together:
```suggestion
            if schema_update_options is None:
                   schema_update_options = []
            else:
                   # To provide backward compatibility
                   schema_update_options = list(schema_update_options)
```

Reply:
Or even more simpler:
```
schema_update_options = list([] or schema_update_options)
```

Reply:
```
schema_update_options = list(schema_update_options or [])
```